File: backend/Models/Rappel.cs
namespace backend.Models;

public class Rappel
{
    public long Id { get; set; }
    public DateOnly DateDebut { get; set; }
    public TimeOnly HeureDebut { get; set; }
    public int MinutesAvantRappel { get; set; }
    public required string Type { get; set; }
    public bool Actif { get; set; }
    public long? MedicamentId { get; set; }
    public long? RendezVousMedicalId { get; set; }
}

File: frontend/proconnectnb/lib/models/rappel.dart
class Rappel {
  final int id;
  final DateTime dateHeure;
  final String type;
  final bool actif;
  final int? medicamentId;
  final int? rendezVousMedicalId;

  Rappel({
    required this.id,
    required this.dateHeure,
    required this.type,
    required this.actif,
    this.medicamentId,
    this.rendezVousMedicalId,
  });
}